pkg/kev/converter/kubernetes/transform_test.go

```diff
@@ -842,15 +860,21 @@ var _ = Describe("Transform", func() {
 		})
 
 		When("project service label instructing to expose the k8s service is specified as comma separated list of domain names", func() {
+			ingressAnnotations := map[string]string{
```

Comment:
I'd skip annotations here too and focus on domain names only.

Reply:
Same as above. I'm happy to re-work the tests but I like that we're validating the `&networkingv1beta1.Ingress` object while ensuring the domain is correct.

Reply:
I'd focus the test around the context and have another case for annotations as well.